### ISSUE TITLE
CSS audit

### ISSUE THREAD

**Issue by davidfarmer** [rbeezer/mathbook#741]
Gather issues to be addressed when refactoring the CSS.
Most of these do not require HTML markup changes, but make particular note of any that do.

1.  Paragraph spacing should be accomplished by making space *before* each p, not after.
More generally, most elements should have no bottom-margin. (make notes of any exceptions.)

2. Do an audit of all uses of "header".  Try to make minimal use of "header" in the CSS,
especially when ".header" is sufficient.

3. Try to remove all mention of h1,...,h6.

4. Try to make things look reasonable without the CSS.  For example, change
em.terminology to b.terminology (the CSS only uses the .terminology .)
h1,...,h6 should already be used appropriately, but check on that.

5. Determine if/when .posterior is used, and compare its usage to that of .solutions .

**Comment by rbeezer** [comment]
1.  PTX `term` is now `b.terminology` (nee em.terminology).  Pushed now.  97be07db8d0ac68904d5af139238a23d43d7159c
2.  PTX `abbr`, `acro`, `init` are `abbr.abbreviation`, `abbr.acronym`, `abbr.initialism` (not new).  I removed "Style Me" comments from HTML output - last one of those expedients.  LaTeX uses the "small-capitals" font, but first forces content to lower-case.  Maybe just a slight font-size reduction will make them less prominent.  Forcing to upper-case would provide consistency.
3.  `b` element inventory:  titlepage and colophon (edition, credits, etc.) have lots of `b` elements and lots of `p` for formatting, including one totally empty `p`.  Advise on who should go first on this one?
4.  `b` element inventory: alert is `b` and `i`.  Do we need/want to change that?
5.  `b` element inventory: used for `console` input.  Also part of a bibliography item (which should get overhauled, so I wouldn't fix now.)  I that covers all of them.
